src/components/settings/SettingsAccessTab/DisableLinkManagementAlertModal.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ActionRow,
+  AlertModal,
+  Button,
+  StatefulButton,
+} from '@edx/paragon';
+import { logError } from '@edx/frontend-platform/logging';
+
+const DisableLinkManagementAlertModal = ({
+  isOpen,
+  onClose,
+  onDisableLinkManagement,
+}) => {
+  const [modalDisableButtonState, setModalDisableButtonState] = useState('default');
+
+  const handleClose = () => {
+    if (onClose) {
+      onClose();
+    }
+  };
+
+  const handleDisableLinkManagement = () => {
+    const disableLinkManagement = async () => {
+      setModalDisableButtonState('pending');
+      try {
+        // TODO: make legit API request
+        await new Promise((resolve) => {
```

Comment:
presuming there is going to be a real api call here soon?

Reply:
That is correct! But, those API endpoints don't yet exist so we're just creating a placeholder for where those API calls might go once they do exist 😄